<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformationMatrixSpinBox</class>
 <widget class="QWidget" name="TransformationMatrixSpinBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>144</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>337</width>
    <height>144</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QDoubleSpinBox" name="dSB03">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-10000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="dSB12">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QDoubleSpinBox" name="dSB00">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="dSB22">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDoubleSpinBox" name="dSB13">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-10000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="dSB32">
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>0.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDoubleSpinBox" name="dSB30">
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>0.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QDoubleSpinBox" name="dSB02">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="dSB23">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-10000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="dSB31">
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>0.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="dSB10">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="dSB21">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="dSB11">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QDoubleSpinBox" name="dSB33">
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="dSB01">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDoubleSpinBox" name="dSB20">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dSB00</tabstop>
  <tabstop>dSB01</tabstop>
  <tabstop>dSB02</tabstop>
  <tabstop>dSB03</tabstop>
  <tabstop>dSB10</tabstop>
  <tabstop>dSB11</tabstop>
  <tabstop>dSB12</tabstop>
  <tabstop>dSB13</tabstop>
  <tabstop>dSB20</tabstop>
  <tabstop>dSB21</tabstop>
  <tabstop>dSB22</tabstop>
  <tabstop>dSB23</tabstop>
  <tabstop>dSB30</tabstop>
  <tabstop>dSB31</tabstop>
  <tabstop>dSB32</tabstop>
  <tabstop>dSB33</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
